common/src/main.c

```diff
@@ -36,6 +45,9 @@ static struct Menu MENU = {
         MENU_ITEM('4', "Performance Counter Tests", perf_test_menu),
         MENU_ITEM('5', "TFLite Unit Tests", tflite_do_tests),
         MENU_ITEM('6', "Benchmarks", do_benchmarks_menu),
+#ifdef PLATFORM_sim
+        MENU_ITEM('x', "Exit (only works in simulation)", exit_sim),
```

Comment:
Consider using Q instead of x to avoid accidental exits.

Reply:
Done.